packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -483,7 +582,10 @@ export class TokenRatesController extends StaticIntervalPollingControllerV1<
    * @returns The controller state.
    */
   async _executePoll(networkClientId: NetworkClientId): Promise<void> {
-    const networkClient = this.getNetworkClientById(networkClientId);
+    const networkClient = this.messagingSystem.call(
```

Comment:
is there a difference between making this call and using this.#getChainIdAndTicker

Reply:
Yes, there is — `#getChainIdAndTicker` doesn't take a network client ID, so we need to resolve that.